auth_user_case_insensitive/README.rst

```diff
@@ -0,0 +1,54 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+=======================
+Case Insensitive Logins
+=======================
+
+This module makes user logins case insensitive. It also overwrites the
+search method to allow these case insensitive logins to work on a database
+that previously had case sensitive logins.
+
+Installation
+============
+
+Install this module as you would any other. No further configuration is
+required.
+
```

Comment:
It is relevant to add that existing logins will all be converted to lowercase upon installation.